Kconfig.sdk-hostap

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2019 - 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "WPA supplicant"
+
+rsource "zephyr/Kconfig"
```

Comment:
I wonder why are there two separate Kconfig files, can't we just keep the configuration in a single file?
I also don't see `zephyt/module.yml` here, which specifies where's module Kconfig and CMakeLists.txt file (see nrfxlib for example: https://github.com/nrfconnect/sdk-nrfxlib/blob/main/zephyr/module.yml)

Reply:
Updated to include module.yml and single KConfig.